README.rst

```diff
@@ -12,9 +12,40 @@
 .. image:: doc/source/_static/gnocchi-logo.png
 
 Gnocchi is an open-source, multi-tenant timeseries, metrics and resources
```

Comment:
In #376, `timeseries` were all changed to `time series`, and I guess that this one got ignored

Reply:
And as Gordon said, the text is also shown in GitHub repo, it might be weird to have '||'  around some words.